client/src/components/ListItem.js

```diff
@@ -38,6 +38,7 @@ export default ListItem;
 ListItem.propTypes = {
```

Comment:
Good job adding prop-types! 

I prefer to have the prop-types at the top of the file, so another dev can quickly understand what kind of props you use, but I know that every dev has a different opinion on this.